impl/src/checks.rs

```diff
@@ -146,9 +214,47 @@ pub fn warn_unused_settings(
 }
 
 #[cfg(test)]
-mod tests {
+pub(crate) mod tests {
   use super::*;
-  use crate::metadata::tests::dummy_raze_metadata;
+  use crate::{metadata::tests::dummy_raze_metadata, settings::tests::dummy_raze_settings};
+  use semver::Version;
+
+  pub fn dummy_modified_metadata() -> Metadata {
+    let mut metadata = dummy_raze_metadata().metadata.clone();
+
+    // Phase 1: Add a dummy dependency to the dependency graph.
```

Comment:
I didn't add that, it's a copy/paste from times past and just hadn't noticed it. I do try to do that for the code I write but I'd prefer to leave this for now since I plan on combing through the tests in https://github.com/google/cargo-raze/issues/310